enrich container id from process id (#15947)
elastic/beats: libbeat/processors/add_process_metadata/docs/add_process_metadata.asciidoc

output, to avoid leaking sensitive data. If `restricted_fields` is `true`, the
field will be present in the output.

`host_path`:: (Optional) By default, the `host_path` field is set to the root

> directory of the host `/`. HostPath is the path where /proc reside. For different 

Reviewer: ```suggestion
directory of the host `/`. This is the path where `/proc` is mounted. For different 
```

> runtime configuration of Kubernetes or docker, the host_path can be set to 

Reviewer: ```suggestion
runtime configuration of Kubernetes or Docker the `host_path` can be set to 
```

> overwrite the default `host_path`

Reviewer: ```suggestion
overwrite the default.
```

`cgroup_prefixes`:: (Optional) By default, the `cgroup_prefixes` field is set to

> `/kubepods` and `/docker`. CgroupPrefix is the prefix where the container id is 

Reviewer: ```suggestion
`/kubepods` and `/docker`. This is the prefix where the container ID is 
```

> inside cgroup. or different runtime configuration of Kubernetes or docker, the 

Reviewer: ```suggestion
inside cgroup. For different runtime configuration of Kubernetes or Docker the 
```

> cgroup_prefixes can be set to overwrite the default `cgroup_prefixes`

Reviewer: ```suggestion
`cgroup_prefixes` can be set to overwrite the defaults.
```

`restricted_fields`:: (Optional) By default, the `process.env` field is not
output, to avoid leaking sensitive data. If `restricted_fields` is `true`, the
field will be present in the output.

`host_path`:: (Optional) By default, the `host_path` field is set to the root
directory of the host `/`. This is the path where `/proc` is mounted. For
different runtime configurations of Kubernetes or Docker, the `host_path` can 
be set to overwrite the default.

`cgroup_prefixes`:: (Optional) By default, the `cgroup_prefixes` field is set
to `/kubepods` and `/docker`. This is the prefix where the container ID is
inside cgroup. For different runtime configurations of Kubernetes or Docker,
the `cgroup_prefixes` can be set to overwrite the defaults.

`cgroup_cache_expire_time`:: (Optional) By default, the 

> `cgroup_cache_expire_time` is set to 5 minuses. This is the length of time

Reviewer 2: it is set to 30 seconds:
https://github.com/elastic/beats/pull/15947/files#diff-2ccbb70ee27fbd9dfb4a9c83b1c00acfR38

Author: Thanks, fixed it

`restricted_fields`:: (Optional) By default, the `process.env` field is not
output, to avoid leaking sensitive data. If `restricted_fields` is `true`, the
field will be present in the output.

`host_path`:: (Optional) By default, the `host_path` field is set to the root
directory of the host `/`. This is the path where `/proc` is mounted. For
different runtime configurations of Kubernetes or Docker, the `host_path` can 
be set to overwrite the default.

`cgroup_prefixes`:: (Optional) By default, the `cgroup_prefixes` field is set
to `/kubepods` and `/docker`. This is the prefix where the container ID is
inside cgroup. For different runtime configurations of Kubernetes or Docker,
the `cgroup_prefixes` can be set to overwrite the defaults.

> `CgroupCacheExpireTime`:: (Optional) By default, the `CgroupCacheExpireTime`

Reviewer 3: ```suggestion
`cgroup_cache_expire_time`:: (Optional) By default, the `cgroup_cache_expire_time`
```
